### PR TITLE
#29 Add SQL Logger

### PR DESCRIPTION
**Pull request by amenk**

### CODE REVIEW

**Comment by shyim** [comment]
Thanks!

File: src/Doctrine/EchoSQLLogger.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+namespace Frosh\DevelopmentHelper\Subscriber;
```
Comment (mynameisbogdan) [inline src/Doctrine/EchoSQLLogger.php:3]: The namespace should be `Frosh\DevelopmentHelper\Doctrine`.

File: README.md [hunk 2]
```diff
@@ -50,3 +50,14 @@ Checks the difference of that definition with the database and creates a migrati
...
+
+### SQL Logger for Console Debugging
+
+Prints executed SQL to the console, in such a way that they can be easily copied to other SQL tools for further
+debugging. This is similar to the symfony debug bar, but useful in CLI commands and tests.
+
+Usage:
+
+     Kernel::getConnection()->getConfiguration()->setSQLLogger(
+         new \Frosh\DevelopmentHelper\Subscriber\EchoSQLLogger()
```
Comment (mynameisbogdan) [inline README.md:62]:
```diff
- new \Frosh\DevelopmentHelper\Subscriber\EchoSQLLogger()
+ new \Frosh\DevelopmentHelper\Doctrine\EchoSQLLogger()
```

File: src/Doctrine/EchoSQLLogger.php (hunk 1 above)
Comment (shyim) [reply src/Doctrine/EchoSQLLogger.php:3]: Can you make a Pull Request?

Comment (mynameisbogdan) [reply src/Doctrine/EchoSQLLogger.php:3]: Done in #31

**Pull request merged**